okio/src/main/java/okio/BufferedSink.kt

```diff
@@ -0,0 +1,358 @@
+   *
+   * assertEquals(8, buffer.size());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0x7f, buffer.readByte());
+   * assertEquals((byte) 0x0f, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals(0, buffer.size());
+   * ```
+   */
+  @Throws(IOException::class)
+  fun writeIntLe(i: Int): BufferedSink
+
+  /**
+   * Writes a big-endian long to this sink using eight bytes.
+   * ```
+   * Buffer buffer = new Buffer();
+   * buffer.writeLong(9223372036854775807L);
+   * buffer.writeLong(15);
+   *
+   * assertEquals(16, buffer.size());
+   * assertEquals((byte) 0x7f, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x0f, buffer.readByte());
+   * assertEquals(0, buffer.size());
+   * ```
+   */
+  @Throws(IOException::class)
+  fun writeLong(v: Long): BufferedSink
+
+  /**
+   * Writes a little-endian long to this sink using eight bytes.
+   * ```
+   * Buffer buffer = new Buffer();
+   * buffer.writeLongLe(9223372036854775807L);
+   * buffer.writeLongLe(15);
+   *
+   * assertEquals(16, buffer.size());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0xff, buffer.readByte());
+   * assertEquals((byte) 0x7f, buffer.readByte());
+   * assertEquals((byte) 0x0f, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals((byte) 0x00, buffer.readByte());
+   * assertEquals(0, buffer.size());
+   * ```
+   */
+  @Throws(IOException::class)
+  fun writeLongLe(v: Long): BufferedSink
```

Comment:
nit: `v` for `value`? why not `l`?

Reply:
I guess.

I think we'll want to make a pass over all the parameter names before releasing since they're technically public API now.

Reply:
Thumbs down on `l`, which looks like `1` and `I`